test/functional/job/job_spec.lua

```diff
@@ -228,10 +228,10 @@ describe('jobs', function()
     it('returns a list of status codes', function()
       source([[
       call rpcnotify(g:channel, 'wait', jobwait([
-      \  jobstart([&sh, '-c', 'sleep 0.10; exit 4']),
-      \  jobstart([&sh, '-c', 'sleep 0.110; exit 5']),
-      \  jobstart([&sh, '-c', 'sleep 0.210; exit 6']),
-      \  jobstart([&sh, '-c', 'sleep 0.310; exit 7'])
+      \  jobstart([&sh, &shellcmdflag, 'sleep 0.10; exit 4']),
+      \  jobstart([&sh, &shellcmdflag, 'sleep 0.110; exit 5']),
+      \  jobstart([&sh, &shellcmdflag, 'sleep 0.210; exit 6']),
+      \  jobstart([&sh, &shellcmdflag, 'sleep 0.310; exit 7'])
```

Comment:
In that case for these tests we should just use powershell. It also has `sleep`.